### ISSUE TITLE
Crash upon launch

### ISSUE THREAD

**Issue by Grimmwyrm** [GTNewHorizons/CodeChickenCore#14]
I am 99.9% positive the issue is me, but I have no idea what I'm doing that is wrong.
I created a modpack for 1.7.10 with Forge 10.13.4.1614
I downloaded CodeChickenCore-1.2.1.jar and CodeChickenLib-1.2.1.jar directly into the mods folder from this site with no other mods.
Each time I launch, I crash near instantly with the error: 

java.lang.NoClassDefFoundError: codechicken/lib/asm/CC_ClassWriter

Looking at my other modpacks, I saw that codechickenlib usually migrated into the folder /mods/1.7.10 so I put dragged CodeChickenLib-1.2.1.jar into it and retried. Same crash. Tried putting both in the 1.7.10 folder. Same error crash.

I am at a loss since I want to use NEI along side Angelica.

Again, I am 99.9% positive the issue is with me and have no idea what I am missing and the only mods in the modpack are CodeChickenCore and CodeChickenLib so its not a mod conflict.

**Comment by Grimmwyrm** [comment]
Update: A Happy One!

I have discovered how not to make it crash.

First I created new modpack and downloaded NotEnoughItems Unofficial which also downloaded the unofficial chickencode/lib. I ran the game and after quitting, replaced the unofficial core/lib with the 1.20+ versions while keeping the NotEnoughItems Unofficial. After adding Unimixins, Angelica, smartrender, and player.api, there were no more crashes.

I don't know why but it must be NEI Unofficial and not NEI for this to work.
